### ISSUE TITLE
RefClassifierTypeSystem.xml - File not found

### ISSUE THREAD

**Issue by erimpapova** [VBRANT/refparse#1]
It seems that there is missing file. By using refparse I get
java.io.FileNotFoundException: .\dataFiles\RefParseData\RefClassifierTypeSystem.xml (The system cannot find the file specified)
after the execution of
myRefParse.setDataProvider(new AnalyzerDataProviderFileBased(myRefParseDataFolder));

Detailed output:
java.io.FileNotFoundException: .\dataFiles\RefParseData\RefClassifierTypeSystem.xml (The system cannot find the file specified)
    at java.io.FileInputStream.open(Native Method)
    at java.io.FileInputStream.<init>(FileInputStream.java:146)
    at de.uka.ipd.idaho.gamta.util.AnalyzerDataProviderFileBased.getInputStream(AnalyzerDataProviderFileBased.java:72)
    at de.uka.ipd.idaho.plugins.bibRefs.BibRefTypeSystem.getInstance(BibRefTypeSystem.java:1228)
    at de.uka.ipd.idaho.plugins.bibRefs.refParse.RefParse.initAnalyzer(RefParse.java:605)
    at de.uka.ipd.idaho.gamta.util.AbstractConfigurableAnalyzer.setDataProvider(AbstractConfigurableAnalyzer.java:327)
    at de.uka.ipd.idaho.plugins.bibRefs.refParse.RefParse.getInstance(RefParse.java:5948)
    at de.uka.ipd.idaho.plugins.bibRefs.refParse.RefParseAnalyzer.initAnalyzer(RefParseAnalyzer.java:40)
    at de.uka.ipd.idaho.gamta.util.AbstractConfigurableAnalyzer.setDataProvider(AbstractConfigurableAnalyzer.java:327)
    at de.uka.ipd.idaho.plugins.bibRefs.refParse.Main.main(Main.java:29)
XsltUtils: testing XSL Transformer Factory for internal caching behavior ...
- got XSL Transformer Factory instance
- got XSL Transformer instance 1
- got XSL Transformer instance 2
- found factory to not be caching internally, can use centralized instance
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:book.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:book.mods'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:book chapter.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:book chapter.mods'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:proceedings.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:proceedings.mods'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:proceedings paper.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:proceedings paper.mods'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:journal volume.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:journal volume.mods'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:journal article.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:journal article.mods'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:url.origin'
  XsltUtils: loaded XSL Transformer Pool from 'DefaultBibRefTypeSystem:url.mods'
  StringUtils: initializing PostScript character mappings ...
  StringUtils: PostScript character mappings initialized.

**Comment by gsautter** [comment]
This exception is merely a report that some custom reference type system could not be loaded for some reason, in which case the default instance will be used. In a sense, this is just kind of a verbose defaulting behavior, not really an error.
It indicates that a mounting point for custom behavior is not in use, or that an attempt to use it failed. The verbosity (stack trace output) is basically a means of helping in the latter case, i.e., that an actual attempt at introducing custom behavior fails for some reason.

In other words, this is nothing to worry about, and as the continuing output tells me, also nothing that would stop RefParse from working. To the contrary, the above output looks like initialization just continued normally.
